README.md

```diff
@@ -30,52 +30,53 @@ _Data is initially pulled from Stanford's system, but the user can update certai
 
 | Column Name | Type | Meaning | Notes |
 | - | - | - | - |
-| `SUNet` | string (primary key) | SUNet id | never shown to users, only used internally |
-| `first_name` * | string | first name |  |
-| `last_name` * | string| last name |  |
-| `email` * | string (email address) | user's email address | assumed initially to be SUNet@stanford.edu, but user can update |
+| `user_id` | string (primary key) | uuid | |
```

Comment:
We should use uuids in the tables. This will protect us against hot spots (not that we'll have millions of users haha but it's a good practice)

Reply:
can you explain what that means?

Reply:
see above! we can discuss more if needed :) again, it doesn't really matter for us lol